release/rllib_tests/learning_tests/hard_learning_tests.yaml

```diff
@@ -456,22 +456,18 @@ sac-halfcheetahbulletenv-v0:
 slateq-interest-evolution-recsim-env:
     env: ray.rllib.examples.env.recommender_system_envs_with_recsim.InterestEvolutionRecSimEnv
     run: SlateQ
-    stop:
+    pass_criteria:
         episode_reward_mean: 162.0
```

Comment:
This won't run using the rllib cli utility -- is that fine?

Reply:
Yeah, that's ok. These are special release tests configs with the additional `pass_criteria` key.